src/include/gurt/telemetry_common.h

```diff
@@ -203,10 +198,18 @@ struct d_tm_histogram_t {
 	int			dth_value_multiplier;
 };
 
+struct d_tm_meminfo_t {
+	uint64_t arena;
+	uint64_t ordblks;
+	uint64_t uordblks;
+	uint64_t fordblks;
+};
+
 struct d_tm_metric_t {
 	union data {
 		uint64_t	value;
 		struct		timespec tms[2];
+		struct d_tm_meminfo_t    meminfo;
```

Comment:
not aligned:)

Reply:
It's a cherry-pick so this should have been addressed on master branch.